superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -437,7 +437,7 @@ const forceSSLField = ({
   changeMethods,
   db,
   sslForced,
-}: FieldPropTypes) => (
+}: FieldPropTypes) => { console.log("db", db); return (
```

Comment:
Remove the console.log and remove the curly brackets here, we won't need any js logic here now that we're done debugging 😁